multiplatform-movies/common/build.gradle

```diff
@@ -1,11 +1,41 @@
 apply plugin: 'kotlin-multiplatform'
 apply plugin: 'kotlinx-serialization'
+apply plugin: 'com.codingfeline.buildkonfig'
+apply plugin: 'com.novoda.build-properties'
 
 // TODO work around for https://youtrack.jetbrains.com/issue/KT-27170
 configurations {
     compileClasspath
 }
 
+buildProperties {
```

Comment:
We use the [build properties plugin](https://github.com/novoda/gradle-build-properties-plugin) to consume the credentials from property files.